docs/website.md

```diff
@@ -65,20 +92,47 @@ The `EMBLEM_API_URL` value will be determined by where you host the Content API.
 
 Congratulations! You are now ready to run the Emblem web app.
 
-## Seeding the Database
-To mimic a real-world production instance, you can deploy the [Content API](../content-api/README.md) and seed the Firestore database with sample data. Add fake campaigns, causes, donors, and donations by running the [`seed_database`](../content-api/data/seed_database.py) script:
-```
-python seed_database.py
-```
+## Running Locally
 
-This script imports content from [`sample_data.json`](../content-api/data/sample_data.json). The campaigns, causes, donors, and donations in the sample data are fictional.
+### Downloading tools
 
-Once the database has been seeded, you can interact with the data on your running Website instance or by making requests to the API directly.
-```
-# Get the URL from your deployed API.
-# If the API is running locally, set EMBLEM_API_URL to your local API URL.
-export EMBLEM_API_URL=$(gcloud run services describe content-api --project $PROJECT_ID --format "value(status.url)")
+To run this project, you'll need the following tools:
 
-# Make an HTTP request to get a cause
-curl -X GET $EMBLEM_API_URL/causes/6aee60eead3741a98f15
-```
+* Python (3.9 or higher)
+* [pip](https://pypi.org/project/pip/)
+
+The following tools are optional, but recommended:
+
+* [pyenv](https://github.com/pyenv/pyenv)
```

Comment:
Nit: we don't refer to this. Can we do one of the following?
1) Explain why it's useful
2) Include an `If you're using Pyenv, ...` blurb

Reply:
This PR is scoped to reorganizing the existing information, PRs to improve the docs are welcome! (I personally have used pyenv once or twice but probably should not be the one to expand on it.)